pkgs/os-specific/linux/kernel/manual-config.nix

```diff
@@ -427,77 +426,80 @@ lib.makeOverridable (
         # which trips the broken symlink check. So, just skip it. We'll know if it explodes.
         dontCheckForBrokenSymlinks = true;
 
-        postInstall = optionalString isModular ''
```

Comment:
Have you researched why this was previously only done for modular builds?

Reply:
My understanding is that `$dev/vmlinux` was added in 53a45825523566b8c9c5ff135e8ff75e05558d85 and that broke non-`isModular` builds because the dev output is conditional on `isModular`. 7ce1f0b8ae74fb7dbaa02bbcbe856026314b8c95 chose to fix this (IIUC, I don't see any specific discussion on the build error) by making `$dev/vmlinux` also conditional, whereas making the dev output unconditional would also work

Reply:
Okay, makes sense. Would be good to note this in the commit message.